packages/babel-plugin-proposal-private-property-in-object/test/fixtures/to-native-fields/accessor/output.js

```diff
@@ -0,0 +1,14 @@
+var _FooBrandCheck = new WeakSet();
```

Comment:
We can also place FooBrandCheck to a private static field, so we avoid the edge cases when `FooBrandCheck` might be injected to an incorrect scope, e.g. when a class is in the param initializer.

```js
// <-- FooBrandCheck will be injected to here
(x = class { #foo; test (other) { return #foo in other } }) => {}
```

Reply:
Ohh I love this idea

Reply:
I wish class access were not stage-2. The class binding can be overwritten. Guess we may have to stick to temporary variable.